internal/kafka/internal/presenters/cluster.go

```diff
@@ -118,15 +117,7 @@ func presentEnterpriseClusterSupportedInstanceTypes(remainingCapacity int32, kaf
 		}, err
 	}
 
-	availableSizes := arrays.Filter(standardInstanceType.Sizes, func(size config.KafkaInstanceSize) bool {
-		return int32(size.CapacityConsumed) <= remainingCapacity
-	})
-
-	presentedSizes := []public.SupportedKafkaSize{}
-
-	if len(availableSizes) > 0 {
-		presentedSizes = GetSupportedSizes(&config.KafkaInstanceType{Sizes: availableSizes})
-	}
+	presentedSizes := GetSupportedSizes(&config.KafkaInstanceType{Sizes: standardInstanceType.Sizes})
```

Comment:
let's also remove the `remainingCapacity` argument as it is not used anywhere in this function https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/6524c0c8ee7217cd4b5e70e2e410b376a242795d/internal/kafka/internal/presenters/cluster.go#L101